container.php

```diff
@@ -196,7 +196,18 @@ public function onRoute(ActionEvent $actionEvent)
             $buildings = $container->get(BuildingRepositoryInterface::class);
 
             return function (Command\RegisterNewBuilding $command) use ($buildings) {
-                $buildings->add(Building::new($command->name()));
+                $buildings->store(Building::new($command->name()));
+            };
+        },
+        Command\CheckInUser::class => function (ContainerInterface $container) : callable {
```

Comment:
The convention of having `'CommandClassName' => $commandHandlerFactory` is defined in the `CommandBus` plugins (same file, almost at the top):

https://github.com/ShittySoft/symfony-live-berlin-2018-cqrs-es-workshop/blob/a73bddb2474e4e9d3a53e5cbb221099a42ba4c94/container.php#L143-L162